.github/workflows/e2e.yml

```diff
@@ -29,10 +31,10 @@ jobs:
     steps:
       - id: prepare
         run: |
-          echo "::set-output name=short_sha::${GITHUB_SHA::7}"
-          echo "::set-output name=pr_title::operator test-e2e-operator (0.0.7-${GITHUB_SHA::7})"
-          echo "::set-output name=oc_project::test-e2e-${GITHUB_SHA::7}"
-          echo "::set-output name=test_pr_base_branch::${GITHUB_SHA::7}-base"
+          echo "::set-output name=short_sha::${{ github.run_id }}"
```

Comment:
The description of run_id says: "A unique number for each workflow run within a repository. This number does not change if you re-run the workflow run." I think we can face the same issue if we re-run the workflow.

Reply:
Also if you use a github id we should not call the variable `short_sha`

Reply:
I was also thinking the same as your first comment, and I came up with the solution to add `github.run_id` along with `github.run_number`. I will update it as well as the variable's name in my next commit.

Reply:
`github.run_attempt` might be more suitable in combination with either `github.run_id` or `github.run_number`. From the [docs](https://docs.github.com/en/actions/learn-github-actions/contexts):

 `gitlab.run_id`
> A unique number for each workflow run within a repository. This number does not change if you re-run the workflow run.

 `gitlab.run_number`
> A unique number for each run of a particular workflow in a repository. This number begins at 1 for the workflow's first run, and increments with each new run. This number does not change if you re-run the workflow run.

`github.run_attempt`
> A unique number for each attempt of a particular workflow run in a repository. This number begins at 1 for the workflow run's first attempt, and increments with each re-run.

Reply:
Oh yes, it was `run_attempt`. Sorry for the confusion.